app/models/user.rb

```diff
@@ -91,6 +111,17 @@ def email_changed?
     authorizations.blank?
   end
 
+  def build_identity_group
+    raise StandardError, "Identity Group Exists" if identity_group
```

Comment:
Should we use a custom error here? Also it looks like this is never tested in the specs (removing this line and all pass).